src/refactorings/change-signature/change-signature.ts

```diff
@@ -114,26 +115,47 @@ function updateCode(
         const args = node.arguments.slice();
         if (args.length) {
           newPositions.forEach((order) => {
-            const arg = node.arguments[order.value.startAt];
-            args[order.value.endAt] = arg;
+            if (order.value.startAt === -1) {
```

Comment:
I guess we can write:

```suggestion
            if (isNewParameter(order)) {
```